operations/availabilityblocks.md

```diff
@@ -66,220 +68,188 @@ Note this operation uses [Pagination](../guidelines/pagination.md) and supports
+| `ServiceIds` | array of string | optional, max 1000 items | Unique identifiers of the [Services](services.md#service) to which [Availability blocks](availabilityblocks.md#availability-block) are assigned. |
+| `AvailabilityBlockIds` | array of string | optional, max 1000 items | Unique identifiers of the requested [Availability blocks](availabilityblocks.md#availability-block). |
+| `CreatedUtc` | [Time interval](_objects.md#time-interval) | optional, max length 3 months | Interval in which the `Availability blocks` were created. |
+| `UpdatedUtc` | [Time interval](_objects.md#time-interval) | optional, max length 3 months | Interval in which the `Availability blocks` were updated. |
+| `CollidingUtc` | [Time interval](_objects.md#time-interval) | optional, max length 3 months | Interval in which the `Availability blocks` are active. |
+| `ReleasedUtc` | [Time interval](_objects.md#time-interval) | optional, max length 3 months | Interval in which the `Availability blocks`are released. |
+| `ExternalIdentifiers` | array of string | optional, max 1000 items | Identifiers of [Availability block](availabilityblocks.md#availability-block)s from external systems. |
+| `States` | array of [Availability block state](availabilityblocks.md#availability-block-state) | optional | States the availability blocks should be in. |
+| `ActivityStates` | array of string | optional | Whether to return only active, only deleted or both records. |
+| `Limitation` | [Limitation](../guidelines/pagination.md#limitation) | required | Limitation on the quantity of data returned. |
 
 #### Availability block extent
+Extent of data to be returned, e.g. it is possible to specify that related service orders (for example reservations) are returned.
 
 | Property | Type | Contract | Description |
 | :-- | :-- | :-- | :-- |
-| `AvailabilityBlocks` | bool | optional | Whether the response should contain the general availability blocks. |
-| `Adjustments` | bool | optional | Whether the response should contain individual availability adjustments related to availability blocks. |
-| `ServiceOrders` | bool | optional | Whether the response should contain reservations related to availability blocks. |
-| `Rates` | bool | optional | Whether the response should contain rates related to availability blocks. |
+| `AvailabilityBlocks` | boolean | optional | Whether the response should contain the general availability blocks. |
+| `Adjustments` | boolean | optional | Whether the response should contain individual availability adjustments related to availability blocks. |
+| `ServiceOrders` | boolean | optional | Whether the response should contain reservations related to availability blocks. |
+| `Rates` | boolean | optional | Whether the response should contain rates related to availability blocks. |
```

Comment:
Make a follow-up or tackle it in this PR.

Let's make `ServiceOrders` and `Rates` deprecated, they are replaced by `reservations/getAll/2023-06-06` and `rates/getAll`. I'm not sure with Adjustments, we do have `availabilityAdjustments/getAll` but it has filtering by `updatedUtc` so getting by `AvailabilityBlocks` is not possible now 🤔

Reply:
done, Ive depricated `ServiceOrders `and `Rates` [here](https://github.com/MewsSystems/gitbook-connector-api/pull/651/commits/fe829f316858ab533447846864640b026c44f4fc)